core/server/data/migration/fixtures/006/01-transform-dates-into-utc.js

```diff
@@ -0,0 +1,201 @@
+                        user.set('last_login', _private.addOffset(user.get('last_login')));
+                    }
+
+                    if (user.get('updated_at')) {
+                        user.set('updated_at', _private.addOffset(user.get('updated_at')));
+                    }
+
+                    user.set('created_at', _private.addOffset(user.get('created_at')));
+
+                    options.id = user.get('id');
+                    return models.User.edit(user.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Users!');
+                });
+            });
+        },
+        function () {
+            return models.Subscribers.forge().fetch().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Subscribers found');
+                    return;
+                }
+
+                return Promise.mapSeries(result.models, function mapper(subscriber) {
+                    if (subscriber.get('unsubscribed_at')) {
+                        subscriber.set('unsubscribed_at', _private.addOffset(subscriber.get('unsubscribed_at')));
+                    }
+
+                    if (subscriber.get('updated_at')) {
+                        subscriber.set('updated_at', _private.addOffset(subscriber.get('updated_at')));
+                    }
+
+                    subscriber.set('created_at', _private.addOffset(subscriber.get('created_at')));
+
+                    options.id = subscriber.get('id');
+                    return models.Subscriber.edit(subscriber.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Subscribers!');
+                });
+            });
+        },
+        function () {
+            return models.Settings.forge().fetchAll().then(function (result) {
+                if (result.models.length === 0) {
+                    logger.warn(messagePrefix + 'No Settings found');
+                    return;
+                }
+
+                return Promise.mapSeries(result.models, function mapper(settings) {
+                    if (settings.get('updated_at')) {
+                        settings.set('updated_at', _private.addOffset(settings.get('updated_at')));
+                    }
+
+                    settings.set('created_at', _private.addOffset(settings.get('created_at')));
+
+                    options.id = settings.get('id');
+                    return models.Settings.edit(settings.toJSON(), options);
+                }).then(function () {
+                    logger.info(messagePrefix + 'Updated datetime fields for Settings!');
+                });
+            });
+        },
+        function () {
+            return Promise.all(['Roles', 'Permissions', 'Tags', 'Apps', 'AppSettings', 'AppFields', 'Clients'].map(function (model) {
```

Comment:
I think this also needs to be done with `mapSeries`? 

`return Promise.mapSeries(['Roles', 'Permissions', 'Tags', 'Apps', 'AppSettings', 'AppFields', 'Clients'], function (model) {...`

This will ensure that the models get updated in the same order each time, and that if a user has, for example, a very large number of tags, it reduces the chance of us having issues.